KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment2/StockServiceFactory.java

```diff
@@ -0,0 +1,18 @@
+
+package edu.KaylaKornelis.advancedjava.Assignment2;
+
+/**
+ *
+ * @author Kornelis
+ */
+public class StockServiceFactory {
+    
+    public StockService getStockService(StockService stockServiceType){
+        if (stockServiceType == null){
+            return null;
```

Comment:
NOT GOOD - never return null unless you have to!

Reply:
Removed the return null;